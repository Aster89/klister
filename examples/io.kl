#lang "prelude.kl"

(example (the Output-Port stdout))

(example (bind-IO (pure-IO 5) (lambda (x) (pure-IO (+ x 3)))))

(run (write stdout "hello world!\n"))
