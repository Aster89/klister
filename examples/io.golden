#<IO action> : (IO Integer)
hello world!
